augur/refine.py

```diff
@@ -241,11 +241,27 @@ def run(args):
     if args.divergence_units=='mutations-per-site': #default
         pass
     elif args.divergence_units=='mutations':
-        L = tt.seq_len
-        for node in node_data['nodes']:
+        if not args.timetree:
+            tt.infer_ancestral_sequences()
+        nuc_map = profile_maps['nuc']
+
+        def different(a,d):
+            '''
+            determine whether two ancestral states should count as mutation for divergence estimates
+            while correctly accounting for ambiguous nucleotides
+            '''
+            if a in ['-', 'N'] or d in ['-', 'N']:
+                return False
+            elif a in nuc_map and d in nuc_map:
+                return np.sum(nuc_map[a]*nuc_map[d])==0
+            else:
+                return False
+
+        for node in T.find_clades():
+            n_muts = len([p for a,p,d in node.mutations if different(a,d)])
```

Comment:
Same request for this line as in the `different` function, but can we use human-readable variable names here? I can push a change for these.